test/unit/util/sfToolkitTest.php

```diff
@@ -10,11 +10,12 @@
 
 require_once __DIR__.'/../../bootstrap/unit.php';
 
-$t = new lime_test(95);
+$t = new lime_test(96);
 
 // ::stringToArray()
 $t->diag('::stringToArray()');
 $tests = array(
+    null => array(),
```

Comment:
This `null` index is converted to an empty string.

So, need to think to test in different way.